src/stdune/sexp.ml

```diff
@@ -235,12 +236,28 @@ module Of_sexp = struct
   let field_missing state name =
     of_sexp_errorf_loc state.loc "field %s missing" name
 
+  let [@inline never] rec multiple_occurrences ~name ~last ~prev =
```

Comment:
As noted in CI, this isn't supported in 4.02. I don't know much about the inliner, but i thought that recursive definitions do not get inlined anyways.

Reply:
I tried to move the attribute after.

> but i thought that recursive definitions do not get inlined anyways.

I think they can with flambda